tests_phi/pages/general_setting_pages.py

```diff
@@ -0,0 +1,34 @@
+from selenpy.element.base_element import BaseElement
+from selenpy.element.text_box import TextBox
+from selenpy.element.combo_box import ComboBox
+import logging
+
+class DAGeneralSettingsPage():
+
+    def __init__(self):
+        self.dynamic_button = BaseElement("//input[@value='%s']")
+        self._txt_name = TextBox("id=txtProfileName")
+        self._cbb_type = ComboBox("id=cbbEntityType")
+        self._cbb_related_data = ComboBox("id=cbbSubReport")
+        
+    def enter_general_setting_info(self, name, item_type = None, related_data = None):
+        self._txt_name.wait_for_visible()
+        self._txt_name.find_element().clear()
```

Comment:
should write clear() in element/textbox method

Reply:
done